Nef_3/include/CGAL/Nef_polyhedron_3.h

```diff
@@ -49,7 +49,7 @@
 #include <CGAL/Nef_3/Mark_bounded_volumes.h>
 
 #include <CGAL/IO/Verbose_ostream.h>
-#include <CGAL/Nef_3/polyhedron_3_to_nef_3.h>
+#include <CGAL/Nef_3/polygon_mesh_to_nef_3.h>
```

Comment:
Nef is already a huge machinery. I don't see including some BGL headers as a problem here.